src/test/test.iml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
what is this file for?

Reply:
oh i think it got added when i was configuring test folder?